data/multi_agency_files.mk

```diff
@@ -1,28 +1,28 @@
 VPATH=data
 
 
-.INTERMEDIATE : 2017-pt-%-with-agencies.csv 2017-pt-%-with-data-year.csv \
-	2017-pt-%-with-valid-start-dates.csv 2017-pt-%-salary-summed.csv \
-	2017-pt-%-no-salary-omitted.csv %-amendment-with-agencies.csv
+.INTERMEDIATE : %-with-agencies.csv %-with-data-year.csv \
+	%-with-valid-start-dates.csv %-salary-summed.csv \
+	%-no-salary-omitted.csv %-amendment-with-agencies.csv
 
-2017-actual-pt-%.csv : 2017-pt-%-no-salary-omitted.csv
+payroll-actual-%.csv : %-no-salary-omitted.csv
+	# Rename fields.
 	(echo employer,last_name,first_name,title,department,base_salary,extra_pay,date_started,id,_,responding_agency,data_year,salary; \
 	tail -n +2 $<) > $@
 
-2017-pt-%-no-salary-omitted.csv : 2017-pt-%-salary-summed.csv
-	csvgrep -c total_pay -r '^$$' -i $< > $@
+%-no-salary-omitted.csv : %-with-valid-start-dates.csv
+	# Remove records where no salary was reported.
+	csvgrep -c base_salary,extra_pay -r '^$$' -i $< > $@;
+	echo "Removed $$(csvgrep -c base_salary,extra_pay -r '^$$' $< | wc -l | xargs) records without salary"
 
-2017-pt-%-salary-summed.csv : 2017-pt-%-with-valid-start-dates.csv
-	cat $< | python data/processors/sum_salary.py > $@
-
-2017-pt-%-with-valid-start-dates.csv : 2017-pt-%-with-data-year.csv
-	# Remove invalid dates. (Only two in the first round.)
+%-with-valid-start-dates.csv : %-with-data-year.csv
+	# Remove invalid dates.
 	cat $< | python data/processors/validate_dates.py > $@
 
-2017-pt-%-with-data-year.csv : 2017-pt-%-with-agencies.csv
+%-with-data-year.csv : %-with-agencies.csv
 	# Add required data year field.
-	perl -pe 's/$$/,2017/' $< > $@
+	perl -pe "s/$$/,$$(cut -d '-' -f 1 <<< $*)/" $< > $@
```

Comment:
Was not able to make the data. My terminal showed the error ending at this line, with this: 

```bash
# Join standard data with agency lookup.
csvjoin -c id,ID -e IBM852 data/raw/payroll-actual-2017-pt-1.csv data/raw/foia-source-lookup.csv > 2017-pt-1-with-agencies.csv
/usr/local/lib/python3.5/site-packages/agate/utils.py:276: UnnamedColumnWarning: Column 9 has no name. Using "j".
# Add required data year field.
perl -pe "s/$/,$(cut -d '-' -f 1 <<< 2017-pt-1)/" 2017-pt-1-with-agencies.csv > 2017-pt-1-with-data-year.csv
/bin/sh: 1: Syntax error: redirection unexpected
make: *** [data/multi_agency_files.mk:24: 2017-pt-1-with-data-year.csv] Error 2
rm 2017-pt-1-with-agencies.csv
```